tests/go.mod

```diff
@@ -0,0 +1,15 @@
+module matter-pi-gpio-commander-tests
+
+go 1.21.6
+
+require github.com/canonical/matter-snap-testing v1.0.0-beta
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/kr/text v0.2.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/testify v1.8.4 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+replace github.com/canonical/matter-snap-testing => github.com/canonical/matter-snap-testing v1.0.0-beta.0.20240208185636-2f282dddac85
```

Comment:
This depends on https://github.com/canonical/matter-snap-testing/pull/11 which needs to merge first.

Reply:
Yes, that's why for now I'm fetching the code from the unmerged branch `exec-context-verbose`, because I need to use on the code some functions that we refactored (e.g. `ExecContextVerbose`).

Reply:
Right. I commented on it only to add visibility into something that needs to be fixed before merging.

Reply:
There has been a bug fix affecting the command with context execution: https://github.com/canonical/matter-snap-testing/pull/11/commits/0e543045d495b431c76f064ef93a302ba31749c0

Reply:
Changed to the latest tag of [matter snap testing](https://github.com/canonical/matter-snap-testing/releases/tag/v1.0.0-beta.1) repository.